CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.0 - 2018-07-12
```

Comment:
Avoid breaking change releases for this type of update. Will require folks to update their pubspecs, which is not needed!